srcpkgs/limine/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'limine'
+pkgname=limine
+version=3.4.4
+revision=1
+archs="x86_64* i686*"
+build_style=gnu-configure
+hostmakedepends="mtools nasm"
+short_desc="Advanced x86/x86_64 BIOS/UEFI bootloader"
+maintainer="Mintsuki <mintsuki@protonmail.com>"
+license="BSD-2-Clause"
+homepage="https://limine-bootloader.org/"
+distfiles="https://limine-bootloader.org/files/limine/limine-${version}.tar.gz"
+checksum=46edf68c36b98a277fb19564b46d295f2f2808dca0e1f01597f1d48709136bb6
+
+if [ "$XBPS_MACHINE" = "i686" ]; then
```

Comment:
what about the case where host=i686, target=i686?

Reply:
Well that `if` should cover both cases, no?